### DISCUSSION TITLE
Can I delete the preinstalled docs directories?

### DISCUSSION THREAD

**Discussion by neonone1234** [thuliteio/doks#1014] [Q&A] [answered]
Hello, I am building a website using the Doks theme and the preinstalled directories don't work for me at all. Is there a way for me to delete them?
Because they appear in the navigation below the docs page like that: 
![Screen Shot 2023-02-25 at 13 33 51](https://user-images.githubusercontent.com/99328716/221352242-7cf855b2-caa5-474f-a58d-f29dfd7b478f.png)

**Comment by h-enk** [comment, marked as answer]
Yes, just delete the content directories and files you don't need

**Reply by neonone1234** [reply]
I just did so but it didn't work.

**Reply by h-enk** [reply]
Did you stop the dev server and run the dev command again? Also, make sure to clear your browser's cache

**Reply by neonone1234** [reply]
Oh, thanks! Sorry for the dumb question 😅
